lib/ffmpex.ex

```diff
@@ -140,14 +140,13 @@ defmodule FFmpex do
 
   Returns `{:ok, output}` on success, or `{:error, {cmd_output, exit_status}}` on error.
   """
-  @spec execute(command :: Command.t) :: :ok | {:error, {Collectable.t, exit_status :: non_neg_integer}}
+  @spec execute(command :: Command.t) :: {:ok, binary()|nil} | {:error, {Collectable.t, exit_status :: non_neg_integer}}
   def execute(%Command{} = command) do
-    {executable, cmd_args} = prepare(command)
+    {executable, args} = prepare(command)
+    cli_command  = [executable | args] |> Enum.join(" ")
+    output = :exec.run(cli_command, [:sync, :stdout, :stderr])
```

Comment:
I think we should pass the command as a list of Erlang strings, aka charlists, to avoid reliance on the shell (and `SHELL` environment variable). Avoiding the shell also avoids shell injection vulnerabilities, according to the documentation of the `cmd()` type: https://saleyn.github.io/erlexec/exec.html

Reply:
fixed, good catch!
I think using `execve` will avoid the need to export the SHELL variable in the CI as well